design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
 
-## Authors
-- Write your design spec on the wiki under the [Documents](/design/specs) folder and link to it from the top-level [Documents](/design/specs) page.
+## Review Process
+
+### Authors
+
+- Clone the [Mathesar Wiki Repo](https://github.com/centerofci/mathesar-wiki) and create a new page under the [Design Specs](/design/specs) directory. Additionally, create a link to it from the top-level [Documents](/design/specs) page.
   - Please follow the format of the most recent spec. Spec formats may change over time.
-- Once you think the spec is ready for review, create a GitHub Discussion that includes:
-  - the design document and all related resources such as wireframes, prototypes, etc. 
+- Add the [spec content](##spec-content) to the page. Make sure the page date is updated as well.
+- Once you think the spec is ready for review, create a PR that includes:
+  - a link to the referenced design issue from the [Mathesar Repo](https://github.com/centerofci/mathesar)
   - an expected timeline for the review, if any.
 - Be sure to include the specific feedback you are seeking.
-- Tag the required reviewers by their GitHub username.
-  - Always tag the Mathesar members with Product or Design roles (see [Team](/team) for handles). If the issue is related to implementation, tag the members with Engineering roles as well.
-- Please post the GitHub discussion link to `#design-review:matrix.mathesar.org`. 
-- If the reviewers request changes or have questions, please make the changes and re-request the review by posting on the GitHub Discussion and tagging reviewers.
-  - Make sure to summarize your changes in your comment.
-- Once everyone has had a chance to review (or the timeline expires) and you're satisfied with the feedback, update the document and ensure that it all fits together well.
-
-[Here's an example review request for reference](https://github.com/centerofci/mathesar/discussions/305).
-
-## Reviewers
-- Check for outstanding design review requests at least once a day.
-- Comment on the GitHub Discussion with feedback. Please follow the general advice above.
-- Once you're done reviewing the document, if you don't have any feedback, post on the discussion and say you're done reviewing.
-- Specify explicitly if you want another look at the design spec before it is finalized.
+- Assign the required reviewers by their GitHub username.
+  - Always request a review from the Mathesar members with Product or Design roles (see [Team](/team) for handles.
+  - If the issue contains technical implementation details, you can request a review from a member with Engineering role as well. Keep in mind that a review from a single Engineering role member should be enough for most design issues. If you are tagging more than one Engineering member, specify the type of feedback that is required from them.
```

Comment:
We decided in yesterday's meeting that we would assign one backend engineer and one frontend engineer (with rotation among backend and frontend engineers) for each spec.